app/assets/stylesheets/components/overview.scss

```
@@ -22,3 +22,13 @@
 .activity-section-alignment {
   margin-left: -15px;
 }
+
+.overview-stats-panel {
+  padding-left: 20px;
+  padding-top: 28px;
+  height: 140px;
+}
+
+.overview-stats-pannel:hover {
```

This has a typo, so not sure how it'd be working...

```suggestion
.overview-stats-panel:hover {
```